README.md

```diff
@@ -4,4 +4,177 @@
 |---|:--:|:--:|
 |[master](https://github.com/dotnet/roslyn-sdk/tree/master)|[![Build Status](https://ci.dot.net/job/Private/job/dotnet_roslyn-sdk/job/master/job/windows_debug//badge/icon)](https://ci.dot.net/job/Private/job/dotnet_roslyn-sdk/job/master/job/windows_debug/)|[![Build Status](https://ci.dot.net/job/Private/job/dotnet_roslyn-sdk/job/master/job/windows_release//badge/icon)](https://ci.dot.net/job/Private/job/dotnet_roslyn-sdk/job/master/job/windows_release/)|
 
-This repository contains code for both the Roslyn-SDK templates and Syntax Vizualizer.
\ No newline at end of file
+This repository contains code for both the Roslyn-SDK templates and Syntax Vizualizer.
+
+# Installation instructions
+
+## Visual Studio 2017
+
+1. Run **Visual Studio Installer**
+2. Hit **Modify**
+3. Select the **Individual components** tab
+4. Check the box for **.NET Compiler Platform SDK**
+
+## Visual Studio 2015
+
+For older versions of Visual Studio the [.NET Compiler Platform SDK](https://visualstudiogallery.msdn.microsoft.com/2ddb7240-5249-4c8c-969e-5d05823bcb89) is available as an extension in the Visual Studio gallery.
+
+# Syntax Visualizer
+
+## Contents
+* [Getting Started](#getting-started)
+* [Syntax Visualizer](#syntax-visualizers)
+* [Syntax Graphs](#syntax-graphs)
+* [Inspecting Semantics](#inspecting-semantics)
+* [Closing and Disabling Syntax Visualizer](#closing-and-disabling-syntax-visualizer)
+
+This document is an overview of the Syntax Visualizer tool that ships as part of the .NET Compiler Platform (“Roslyn”) SDK. The Syntax Visualizer facilitates inspection and exploration of Roslyn syntax trees and can be used as a debugging aid when you develop your own applications atop the .NET Compiler Platform (“Roslyn”) APIs.
+
+## Getting Started
+
+To understand the concepts presented in this document better, I would strongly recommend reading the [.NET Compiler Platform (“Roslyn”) Overview document](https://github.com/dotnet/roslyn/wiki/Roslyn%20Overview). This document provides a very good overview of the .NET Compiler Platform (“Roslyn”) APIs including syntax trees, nodes, tokens, and trivia.
+
+## Syntax Visualizer
+Once installed, the Syntax Visualizer enables inspection of the syntax tree for any C# or VB code file that is open inside the Visual Studio IDE. The visualizer can be launched by clicking on View –> Other Windows –> Roslyn Syntax Visualizer.
+
+*Figure 1 Launching the Syntax Visualizer*
+
+![fig1.png](images/fig1.png)

```

Comment:
❗️ All images are missing from the PR

Reply:
I know. I wasn't sure if you are willing to accept the PR and even if you do it's still debatable if the documentation

* Should be all in readme.md
* Should be in a docs folder and readme.md only has the installation instructions and points to the docs in the docs folder
* Should be in the wiki

If it's all in the readme.md I can also copy the images.

Reply:
Do you believe you would have found it if the documentation was here? I think I still would have had to ask a question.

Reply:
📝 Note that I'm not saying this is good or bad. Perhaps it's something that works for everyone else, or maybe it's just a step in the right direction that needs other changes as well.

Reply:
There are several issues open in roslyn and here that describe the situation and give instructions how to solve it. All of them have at least one thumb up. If people are able to find it in the issues it's clear to me many more will find it in the readme.

Reply:
The major issue is that the documentation in the roslyn wiki and on marketplace are out of date and it is mentioned nowhere how to get the syntax visualizer (even you needed to ask). I just wanted to help out with updating the docs and as I'm not able to contribute to the roslyn wiki and this is the better place anyway I copied the documentation and actualized the installation instructions. I think there is nothing wrong with accepting the PR and dropping it in favor of something else (docs.microsoft.com?) later.

Reply:
> or maybe it's just a step in the right direction that needs other changes as well

There are only two places where the syntax visualizer is mentioned. These two places need to be updated:

* [roslyn wiki](https://github.com/dotnet/roslyn/wiki/Syntax%20Visualizer)
* [Marketplace](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.NETCompilerPlatformSDK)